course-material/references.bib

```diff
@@ -0,0 +1,56 @@
+---
+---
+
+@inproceedings{holdgraf_evidence_2014,
```

Comment:
sample file

Reply:
removed all sample files in latest commit